web_widget_many2one_simple/README.rst

```diff
@@ -61,6 +61,9 @@ Options:
 Usage
 =====
 
+Go to Contacts > Create or edit a record and set "Industry" field (in Sales & Purchase tab).
```

Comment:
Put this after the other paragraph indicating that using demo data, the widget can be used this way.